tests/test_js.py

```diff
@@ -1577,3 +1577,22 @@ async def error_handler(e):
 
         with pytest.raises(BadBucketError):
             await js.key_value(bucket="TEST3")
+
+class ConsumerReplicasTest(SingleJetStreamServerTestCase):
+    @async_test
+    async def test_number_of_consumer_replicas(self):
+        nc = await nats.connect()
+
+        js = nc.jetstream()
+        await js.add_stream(name="TESTREPLICAS", subjects=["test.replicas"])
+        for i in range(0, 10):
+            await js.publish("test", f'{i}'.encode())
```

Comment:
so that it receives messages, has to be following subject:
```suggestion
            await js.publish("test.replicas", f'{i}'.encode())
```

Reply:
Thanks. Fixed the subject name